.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
```

Comment:
Sorry, i didn't know you already use circle-ci2. I would try to change it to be like other one.